src/cli.ts

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+import * as program from 'commander';
+import {startServer} from './server/server';
+import {CLIArguments, BuildCommandArgs} from './types';
+import {createZipProvider} from './scripts/createProviderZip';
+import {executeWebpack} from './webpack/webpack';
+
+/**
+ * Start command
+ */
+program.command('start')
+    .option(
+        '-v, --providerVersion <version>',
+        'Sets the runtime version for the provider.  Defaults to "local". Options: local | staging | stable | w.x.y.z',
+        'local')
+    .option('-m, --mode <mode>', 'Sets the webpack mode.  Defaults to "development".  Options: development | production | none', 'development')
+    .option('-l, --noDemo', 'Runs the server but will not launch the demo application.', true)
```

Comment:
Can we rename the short flag?

`-n`? `-d`?..

Reply:
Going with `-n`